python/foglamp/plugins/south/pt100/pt100.py

```diff
@@ -140,10 +149,11 @@ def plugin_reconfigure(handle, new_config):
 
     # Plugin should re-initialize and restart if key configuration is changed
     if 'pollInterval' in diff:
-        new_handle = copy.deepcopy(new_config)
-        new_handle['restart'] = 'no'
+        plugin_shutdown(handle)
+        new_handle = plugin_init(new_config)
```

Comment:
Do we really need to restart/shutdown? As this plugin is a POLL mode. I think below will work
```
new_handle = copy.deepcopy(handle)
new_handle['restart'] = 'no'
```